modules/javafx.graphics/src/main/java/javafx/css/StyleableDoubleProperty.java

```diff
@@ -148,5 +151,17 @@ public void onStop() {
         public StyleableProperty<?> getStyleableProperty() {
             return StyleableDoubleProperty.this;
         }
+
+        @Override
+        public boolean updateReversingAdjustedStartValue(TransitionMediator existingMediator) {
+            var mediator = (TransitionMediatorImpl)existingMediator;
```

Comment:
A reversing mediator must be the same type, otherwise something is seriously broken. I prefer a `ClassCastException` to highlight that this is very unexpected.